lib/page/countries.rb

```diff
@@ -0,0 +1,18 @@
+require 'everypolitician'
+require_relative '../world'
+
+module Page
+  class Countries
+    def countries
+      @countries ||= EveryPolitician.countries
+    end
+
+    def download_url
+      @download_url ||= EveryPolitician.countries_json
+    end
+
+    def world
+      @world ||= World.new.as_json.to_a
```

Comment:
Ah, I had the same thought, but I just copied what was in the route. Shame on me!